doc/nrf/known_issues.rst

```diff
@@ -489,15 +490,15 @@ KRKNWK-6073: Potential delay during FOTA
 Matter (Project CHIP)
 =====================
 
-.. rst-class:: v1-6-1 v1-6-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0
 
 KRKNWK-10589: Android CHIPTool crashes when commissioning a Matter device
   In random circumstances, Android CHIPTool crashes when trying to connect to a Matter device over Bluetooth® LE.
 
   **Workaround** Restart the application and try to commission the Matter device again.
   If the problem persists, clear the application data and try again.
 
-.. rst-class:: v1-6-1 v1-6-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0
```

Comment:
It is no longer an issue in 1.7, but I'm not sure if we should update it here or in a separate PR (?)

Reply:
It's enough not to add the tag in this PR.